### PR TITLE
fix Admission Webhooks configuration example

### PR DESCRIPTION
**Pull request by CaoShuFeng**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> For 1.11 Features: set Milestone to 1.11 and Base Branch to release-1.11
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> NOTE: After opening the PR, please *un-check and re-check* the ["Allow edits from maintainers"](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) box so that maintainers can work on your patch and speed up the review process. This is a temporary workaround to address a known issue with GitHub.> 
>
> Please delete this note before submitting the pull request.

Allow edits from maintainers checkbox

### CODE REVIEW

File: content/en/docs/reference/access-authn-authz/extensible-admission-controllers.md [hunk 1]
```diff
@@ -176,14 +176,18 @@ The schema of `admissionConfiguration` is defined
 apiVersion: v1
 kind: Config
 users:
-# DNS name of webhook service, i.e., <service name>.<namespace>.svc, or the URL
```
[the remark this answers is not in the document]
Comment (CaoShuFeng) [reply content/en/docs/reference/access-authn-authz/extensible-admission-controllers.md]: The related codes:
https://github.com/kubernetes/kubernetes/blob/def5179c661df6f90e47a37a698f8d1f56acd05d/staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/config/client.go#L177

https://github.com/kubernetes/kubernetes/blob/def5179c661df6f90e47a37a698f8d1f56acd05d/staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/config/authentication.go#L89

[the remark this answers is not in the document]
Comment (CaoShuFeng) [reply content/en/docs/reference/access-authn-authz/extensible-admission-controllers.md]: Pending for a while.
Ping @deads2k for a tech review.
Thanks.

File: content/en/docs/reference/access-authn-authz/extensible-admission-controllers.md [hunk 2]
```diff
@@ -176,14 +176,18 @@ The schema of `admissionConfiguration` is defined
...
     client-certificate-data: <pem encoded certificate>
     client-key-data: <pem encoded key>
 # The `name` supports using * to wildmatch prefixing segments.
-- name: '*.webhook-company.org'
+- name: '*.webhook-company.org'   # This matches webhook URL https://webhook1.webhook-company.org/validatePods
+  user:
+    password: <password>
+    username: <name>
+# `port` is requried when the webhook url contains a port
+- name: '*.webhook-company.org:443'  # This matches webhook URL https://webhook1.webhook-company.org:443/validatePods
```
Comment (deads2k) [inline content/en/docs/reference/access-authn-authz/extensible-admission-controllers.md]: This is actually side-effect, not strictly intent.  It's neat that it works though.  If you write a test to pin the behavior in the server I can see this as a valuable feature.

Comment (CaoShuFeng) [reply content/en/docs/reference/access-authn-authz/extensible-admission-controllers.md]: Will work on it.
Thanks.

**Comment by chenopis** [comment]
@CaoShuFeng What is the status of this PR?

**Comment by CaoShuFeng** [comment]
> @CaoShuFeng What is the status of this PR?

I would say this depends on: https://github.com/kubernetes/kubernetes/pull/66104

**Comment by chenopis** [comment]
@CaoShuFeng So merge this PR once https://github.com/kubernetes/kubernetes/pull/66104 has been merged?

**Comment by zacharysarah** [comment]
**Note:** Docs reviewers, this PR has been reviewed and approved pending the merge of https://github.com/kubernetes/kubernetes/pull/66104. 

@CaoShuFeng 👋 It looks like https://github.com/kubernetes/kubernetes/pull/66104 is stalled (last change 22 days ago). Here's what I'd like to do:

1. Close this PR pending merge of https://github.com/kubernetes/kubernetes/pull/66104.

I've added a note to this PR description so that when https://github.com/kubernetes/kubernetes/pull/66104 merges, this PR can be merged quickly.

2. Remove the `/approve` pending merge of https://github.com/kubernetes/kubernetes/pull/66104. 

Right now, the only thing preventing this PR merging to k/website is a `/hold` that anyone can remove, regardless of whether the dependent PR ☝️ is merged or not.

3. Note that this PR is ready to merge when its dependency merges.

I've added notes to the original description and labels to flag this PR as ready for merge to other reviewers.

**Pull request closed without merging**